hand.py

```diff
@@ -0,0 +1,160 @@
+import cv2
+import mediapipe
+import pyautogui
+import random
+import os
+import pyscreeze
+
+# Initialize Hand tracking module
+capture_hands = mediapipe.solutions.hands.Hands()
+drawing = mediapipe.solutions.drawing_utils
+
+# Get screen dimensions
+screen_height, screen_width = pyautogui.size()
+
+window_width = int(screen_width * 1.0)
+window_height = int(screen_height * 0.4)
+
+# Open webcam
+cam = cv2.VideoCapture(0)
+
+# Initialize variables for hand landmarks
+x1 = y1 = x2 = y2 = 0
+use_two_fingers = True  # Initially use two fingers (index and thumb)
+
+# Variable to track whether the screenshot has been captured
+screenshot_captured = False
+
+#Increase window dimensions to default size
+cv2.namedWindow("Hand gesture", cv2.WINDOW_NORMAL)
+cv2.resizeWindow("Hand gesture", window_width, window_height)
+
+while True:
+    _, img = cam.read()  # Read a frame from the webcam
+    img = cv2.flip(img, 1)
+    img_height, img_width, _ = img.shape
+    rgb_img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+    
+    # Process the image to detect hands
+    output_hands = capture_hands.process(rgb_img)
+    all_hands = output_hands.multi_hand_landmarks
+    
+    if all_hands:
+        for i, hand in enumerate(all_hands):
+            # Generate random color for the hand
+            color = (random.randint(0, 255), random.randint(0, 255), random.randint(0, 255))
+            
+            # Draw landmarks on the hand with random color
+            drawing.draw_landmarks(img, hand, mediapipe.solutions.hands.HAND_CONNECTIONS, landmark_drawing_spec=drawing.DrawingSpec(color=color, thickness=2, circle_radius=4))
+            
+            # Get landmarks for one hand
+            one_hand_landmarks = hand.landmark
+            
+            for id, landmark in enumerate(one_hand_landmarks):
```

Comment:
Avoid using 'id' as a variable name since it's a built-in function in Python [medium]

- Generated by [LGTM](https://lgtm.pro)